docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,111 @@
+---
+title: The newest additions to MUI X v6
+description: Support for time zones, a new filtering UI, charts! and more
+date: 2023-07-30T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/card.png" alt="" style="margin-bottom: 16px;" width="2400" height="559" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.0](https://github.com/mui/mui-x/releases/tag/v6.10.0), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Date Pickers](#date-pickers)
+  - [Support for Time Zones](#support-for-time-zones)
+  - [Digital Clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and Paste](#copy-and-paste)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+  - [Filter on Column Headers](#filter-on-column-headers)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+  - [Performance Enhancements: Smoother Scrolling and Expanding Rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - Alpha Version](#charts---alpha-version)
+- [What's next](#whats-next)
+
+## Date pickers
+
+### Support for time zones
+
+No more hassle displaying dates on the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about the conversion logic and keeping the stored value consistent with your backend.
+
+**[live demo showcasing stored date and toggle between system/UTC/timezone]**
```

Comment:
Will this be a gif or playable video? 

Side-note: I could make a screen-recording of this new feature!

Reply:
playable video, like this one for example: https://mui.com/blog/mui-x-v6/#cell-selection